src/workloads/execution/locf.yml

```diff
@@ -0,0 +1,295 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of $locf. The benchmark operations test numeric,
+  string, date and randomized data types. There is a test without partitions, with a
+  single partition, and with multiple partitions for each data type.
```

Comment:
```suggestion
  single partition, and with multiple partitions for each data type. Partitions require carry forward
  to be done individually on each partition, which requires additional tracking and may make queries
  slower.
```
Does this answer the question? Or is it still slightly too vague?

Reply:
That helps some, but I still don't think I know what a partition is here? 

The number of carry forwards is proportional to the number of partitions?

Reply:
Ah, could you just link to https://docs.mongodb.com/manual/reference/operator/aggregation/setWindowFields/ 
That explains what a partition is. There's a good chance someone from outside of query will need to look at a regression in these tests at some point, so that will help. 

And please include Ted's addition too.

Reply:
done